settingtypes.txt

```diff
@@ -3,6 +3,9 @@
 # Static paths do not work well with settings
 #areas.filename (Configuration file path) string (world_path)/areas.dat
 
+# When enabled, makes PvP opt-out rather than opt-in.
```

Comment:
Can the wording be easier for better understanding by non-English-speaking players?

Reply:
I could make it simpler but I feel like it'd lose some nuance and not be completely accurate.

Maybe "Turn PvP on everywhere" would be simple enough? But it doesn't explain how to turn it back off, I can't figure out how to succinctly explain it without using "opt-out" or some similar word

Reply:
I believe that  "Turn PvP on everywhere" is most understandable